pages/cart-management/cart-management.css

```diff
@@ -0,0 +1,158 @@
+@import url(/main.css);
+
+
+:root {
+  --light-grey: #9ca3af;
+  --button-hover: #be185d;
+  --text-green: #16a34a;
+}
+
+.main-container {
+  display: flex;
+  margin-top: 2rem;
+  gap: 5rem;
+  justify-content: space-evenly;
+}
+
+.item-heading {
+  text-decoration: underline var(--secondary-color);
+  margin-top: 5rem;
+  margin-left: 1.3rem;
+  font-size: 1.2rem;
+}
+
+.product-card {
+  display: flex;
+  margin-top: 2rem;
+  border: 1px solid;
+  gap: 6rem;
+  height: 13rem;
+  box-shadow: 0px 0px 5px 5px var(--light-grey);
```

Comment:
You are using the same value in price-details class. You can create a variable if you are using this value multiple places.